myapp/app/models/task.rb

```diff
@@ -0,0 +1,13 @@
+class Task < ApplicationRecord
+
+  STATUSES = ["Not Started", "In Progress", "Completed"].freeze
+
+  belongs_to :user
+  belongs_to :user, :column :assigned_user_id
```

Comment:
I don't know `column` param(and can't find document)
I think you can modify like this
```
belongs_to :assigned_user_id, class_name: 'User', foreign_key: 'user_id'
```

Reply:
Could you give me the url that you found it.
`belongs_to :assigned_user_id, class_name: 'User', foreign_key: 'user_id'`
Its not actually working in my console.

Reply:
I found this url
https://guides.rubyonrails.org/association_basics.html#the-belongs-to-association:~:text=For%20example%2C%20consider%20the%20following%20model%20declarations%3A

Reply:
got a workaround, thanks for the clue ;)